tests/helpers.py

```diff
@@ -5,7 +5,9 @@
 import sys
 import platform
 
-PATH_TO_DEFAULT_BINARY = "/tmp/default.out"
+TMPDIR = os.getenv("TMPDIR", "/tmp")
+DEFAULT_TARGET = f"{TMPDIR}/default.out"
```

Comment:
Since you're improving it, I think we can take that chance to go full Python3 and replace all those paths with proper `pathlib.Path` objects, which would make the type hints more explicit, and lighten the syntax for instance here would be like 
```python
TMPDIR : pathlib.Path = pathlib.Path( tempfile.gettempdir() )
DEFAULT_TARGET : pathlib.Path = TMPDIR / "default.out"
```

This way the `target` argument would also be a path we can easily define, check etc. 

This shouldn't be a big change to your PR (mostly Ctrl+F / replace)

Reply:
done